ts/packages/cli/src/base.ts

```diff
@@ -21,7 +21,7 @@ type BaseConfig = Config & {
   cli: {
     file?: string; // .configu file
     data: Partial<{
-      stores: Record<string, { type: string; configuration: Record<string, any> }>;
+      stores: Record<string, { type: string; configuration: Record<string, any>; cache?: boolean | string }>;
       schemas: Record<string, string>;
       scripts: Record<string, string>;
     }>;
```

Comment:
Take a look at the [product comment here](https://github.com/configu/configu/issues/370#issuecomment-1983870293). The current implementation does not reflect it.

Reply:
again